src/ZPublisher/HTTPResponse.py

```diff
@@ -790,7 +800,7 @@ def _traceback(self, t, v, tb, as_html=1):
     def _error_html(self, title, body):
         return ("""<!DOCTYPE html><html>
   <head><title>Site Error</title><meta charset="utf-8" /></head>
-  <body bgcolor="#FFFFFF">
+  <body bgcolor='#FFFFFF'>
```

Comment:
Why did you change the quotes? That should not be necessary. For consistency's sake please leave the double quotes.

Reply:
I did this because the double quotes confused the XEmacs syntax highlighting (an XEmacs bug) and rendered the code unreadable for me (following code was represented as string and vice versa).

The change is **inside** a string (destined as HTML fragment). Is quote consistency in the generated HTML really a major concern for you?

Reply:
I have a string dislike when it comes to single quotes in HTML tags. But it looks like you have a good reason, so never mind. Try a better editor 😉

Reply:
I have reverted the corresponding commit (hoping that I will not need to work a lot in the code).